content/en/docs/Configuration/api-manager.md

```diff
@@ -18,43 +18,45 @@ This page is under development
 
 ## Email templates override
 
-Emails are sent to API developpers when they register, activate their account, or change password.
+Emails are sent to API developers when they register, activate their account, or change their password.
+
 ![Welcome email](/Images/welcome-email.png)
-You can customize the email templates used for developer notifications in `/opt/Axway/apigateway/system/conf/apiportal/email` folder of the apimngr pod.
+
+You can customize the email templates used for developer notifications in the  `/opt/Axway/apigateway/system/conf/apiportal/email` folder of the apimngr pod.
 
 ## API Management
 
-Upon solution deployment, several Open Banking APIs are deployed and publish in the catalog.
-You can us the admin interface to update them:
+Upon solution deployment, several Open Banking APIs are deployed and published in the catalog.
+You can use the admin interface to update the APIs.
 
 ![open banking apis](/Images/api-manager-apis.png)
 
-These frontend API changes can be done directly on published API:
+These frontend API changes can be done directly on the published API.
 
-* change API logo
-* change API summary
-* change API documentation
+* API logo
+* API summary
+* API documentation
 
-These Frontend API changes can be done only by unpublishing the API first:
+These frontend API changes can be done only by unpublishing the API first.
```

Comment:
Should this be changed to say -  Unpublish the front-end API before applying changes to the API?